README.md

```diff
@@ -16,6 +16,10 @@
 - Git & Github
 
 ## Live Demo
+[Application](https://legendary-pie-bc7c45.netlify.app/)
+
+## Project Documentation
+[Project Video Description](https://www.loom.com/share/7d360a2872884e95a905fc69192e1289)
```

Comment:
- Good job with the video, however, while you've mentioned a part in the project that you're proud of, the video is still missing a part in which you should show a `piece of code` that you're proud of. As this is part of the project requirement. Please, update this missing part of the project requirement.

Reply:
Thank you for the feedback @zieeco 
The mentioned issue has been resolved in the video.